code/modules/events/devil.dm

```diff
@@ -2,6 +2,7 @@
 	name = "Create Devil"
 	typepath = /datum/round_event/ghost_role/devil
 	max_occurrences = 0
```

Comment:
now I'm no coder, but wouldnt a max occurrence of 0 mean there it is impossible for this event to happen

Reply:
fuck